SingularityService/src/main/java/com/hubspot/singularity/data/history/MySQLHistoryJDBI.java

```diff
@@ -18,11 +18,13 @@
 
 public interface MySQLHistoryJDBI extends AbstractHistoryJDBI {
 
-  @SqlUpdate("INSERT INTO requestHistory (requestId, json, createdAt, requestState, user, message) VALUES (:requestId, :json, :createdAt, :requestState, :user, :message)")
+  @SqlUpdate("INSERT INTO requestHistory (requestId, json, createdAt, requestState, user, message) VALUES (:requestId, :json, :createdAt, :requestState, :user, :message) " +
+      "ON DUPLICATE KEY UPDATE json = :json, requestState = :requestState, user = :user, message = :message")
```

Comment:
this is causing tests to fail but so far can't figure out what's wrong with this query